docs/manual/developer/04_style_guide.md

```diff
@@ -174,11 +174,9 @@ and keep these guidelines in mind when writing new code.
     * We use [Code Climate](https://codeclimate.com/quality) to help automate the checking for PEP 8 issues.
     * We do make one change from PEP 8; our maximum line length is 99 characters
 * Methods should be defined before they are called
-* The files in the build system shall be Python 2.7 and Python 3 compatible
```

Comment:
I think we should define the minimal Python version that we need to be compatible with.